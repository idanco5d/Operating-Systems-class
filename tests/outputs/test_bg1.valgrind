==27780== Memcheck, a memory error detector
==27780== Copyright (C) 2002-2022, and GNU GPL'd, by Julian Seward et al.
==27780== Using Valgrind-3.20.0 and LibVEX; rerun with -h for copyright info
==27780== Command: /mnt/c/Users/illus/os/hw/1/smash
==27780== Parent PID: 27763
==27780== 
==27780== 
==27780== FILE DESCRIPTORS: 4 open (3 std) at exit.
==27780== Open file descriptor 3: /mnt/c/Users/illus/os/hw/1/tests/outputs/test_bg1.valgrind
==27780==    <inherited from parent>
==27780== 
==27780== 
==27780== HEAP SUMMARY:
==27780==     in use at exit: 0 bytes in 0 blocks
==27780==   total heap usage: 34 allocs, 34 frees, 79,954 bytes allocated
==27780== 
==27780== All heap blocks were freed -- no leaks are possible
==27780== 
==27780== For lists of detected and suppressed errors, rerun with: -s
==27780== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
