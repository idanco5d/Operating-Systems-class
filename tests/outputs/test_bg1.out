smash> smash: got ctrl-Z
smash: process 8862 was stopped
smash> [1] sleep 100 : 8862 0 secs (stopped)
smash> sleep 100 : 0
smash> [1] sleep 100 : 8862 0 secs
smash> smash: sending SIGKILL signal to 1 jobs:
8862: sleep 100
